vms/administrator/models.py

```diff
@@ -21,7 +21,7 @@ class Administrator(models.Model):
     address = models.CharField(
         max_length=75,
         validators=[
-            RegexValidator(r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\-)|(\.)|(,)|(\:)]+$', ),
+            RegexValidator(r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\-)|(\.)|(,)|(\:)|(@_!#$%^&*()<>?/\|}{~:)]+$', ),
```

Comment:
This is too extensive, I think @keshakaneria's exact list on the issue description is good enough, please follow that, I don't think I have seen `$`, `%` in any address.

Reply:
done
kindly check